Docs/sphinx_documentation/source/BuildingAMReX.rst

```diff
@@ -626,53 +630,53 @@ A list of AMReX component names and related configure options are shown in the t
    +==============================+=================+
    | DIM                          | 1D, 2D, 3D      |
    +------------------------------+-----------------+
-   | ENABLE_DP                    | DP              |
+   | AMReX_PRECISION              | DOUBLE, SINGLE  |
```

Comment:
Do component  names for `find_package(AMReX COMPONENTS ...)` need an `AMReX_` prefix? I think that would be unusual.

Reply:
Oh wait, I was reading the wrong column :see_no_evil: